build.sbt

```diff
@@ -203,6 +218,7 @@ lazy val apiModels = {
   import ReleaseStateTransformations._
   Project("api-models", file("api-models")).settings(Seq(
     name := "mobile-notifications-api-models",
+    publish / skip := false,
```

Comment:
This is unusual, but we've made `publish / skip := true` the default in this project as so many projects are not ones we want to publish.